icp-deploy.tf

```diff
@@ -61,14 +65,22 @@ module "icpprovision" {
       #"docker_password"                 = "${local.registry_creds != "" ? "${replace(local.registry_creds, "/.*:/", "")}" : "" }"
     }
 
+    hooks = {
```

Comment:
I suspect we don't really want to push out this code to main repo. It looks suspiciously like some code we temporarily tried in the lab when we were debugging our storage issues, and I don't think we're suggesting everyone should update their docker outside of the normal procedures for handling docker we already have in place.